### PR TITLE
Fix lint errors

### PR DESCRIPTION
**Pull request by tylersticka**
## Overview

Fixes lint errors introduced between #506 and #507 being merged.

## Testing

Verify status checks pass.

### CODE REVIEW

File: src/demo/button/button.scss [hunk 1]
```diff
@@ -3,11 +3,11 @@
 
 .demo-button {
   color: colors.$primary-brand;
-  transition: transform motion.$speed-quick motion.$ease-out;
 
   // TODO: These font styles are temporary, for demo purposes
   font-family: inherit;
   font-size: 18px;
+  transition: transform motion.$speed-quick motion.$ease-out;
```
Comment (spaceninja) [inline src/demo/button/button.scss]: Is transition temporary? If not, I'd recommend changing the temporary comment to a footnote style to make it clearer which lines it applies to.

Comment (tylersticka) [reply src/demo/button/button.scss]: Everything in `demo` is temporary.

Comment (spaceninja) [reply src/demo/button/button.scss]: If it's temporary, then you're good to go. :)

Comment (spaceninja) [reply src/demo/button/button.scss]: I just wanted to make sure that rule moving below the comment wasn't inadvertently confusing.

**Pull request merged**